tests/test_sphericaldf.py

```diff
@@ -1722,6 +1722,175 @@ def test_constantbeta_differentpotentials_dens_directint():
                              bins=11)
     return None
 
+################# INTERPOLATED POTENTIALS IN DFS ##############################
+
+# Eddington DFs with interpolated potentials
+def test_eddington_interpolatedpotentials_dens_directint():
+    # Some potentials
+    pots= [potential.HernquistPotential(amp=1.3,a=0.8),
+           potential.PlummerPotential(amp=2.3,b=1.3),
+           potential.NFWPotential(amp=1.3,a=1.5),
+           ]
+    tols= [1e-3 for pot in pots]
+    rmins = [0.1,0.2,0.2]
+    for pot,tol,rmin in zip(pots,tols,rmins):
+        print(pot)
+        ipot= potential.interpSphericalPotential(rforce=pot,
+            rgrid=numpy.geomspace(0.01,100.,10001))
```

Comment:
```suggestion
            rgrid=numpy.geomspace(0.001,100.,10001))
```
Using a minimum r of 0.001 seems to be more robust (some of your tests fail for me locally when using 0.01).